twitter/View Controllers/TimelineViewController.m

```diff
@@ -27,21 +32,72 @@ - (IBAction)didTapLogout:(id)sender {
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
     // Get timeline
     [[APIManager shared] getHomeTimelineWithCompletion:^(NSArray *tweets, NSError *error) {
         if (tweets) {
             NSLog(@"😎😎😎 Successfully loaded home timeline");
-            for (NSDictionary *dictionary in tweets) {
-                NSString *text = dictionary[@"text"];
-                NSLog(@"%@", text);
+            self.arrayOfTweets = [[NSMutableArray alloc] init];;
+            for (Tweet *tweet in tweets) {
```

Comment:
We are receiving an array. I think we should try to avoid the loop here

Reply:
Should I cast the NSArray to a NSMutable array and set them equal to each other directly then?